Python_env/hurrevac_storms.py

```diff
@@ -257,27 +259,49 @@ def GetStormJSON(self, StormId):
 
         Keyword Arguments:
            StormId :string -- Hurrevac stormid
+
+        Notes: 
+            https://hvx.hurrevac.com/hvx-api/v1/advisories/storm/{stormidhere has storms and
+            empty json for simulated storms.
+            https://hvx.hurrevac.com/hvx-api/v1/sim/advisories/storm/{stormidhere} has simulated storms and
+            empty json for non-simulated storms.
         """
 ##        logging.debug("Running GetStormJSON")
```

Comment:
minor nit: why are the commented-out lines still here? That could cause some confusion down the road, plus, it makes the code less clean. Can we remove the commented-out lines?

Reply:
Yes, I've removed some commented-out lines.